app/src/main/java/com/github/sdpteam15/polyevents/helper/DatabaseHelper.kt

```diff
@@ -0,0 +1,124 @@
+package com.github.sdpteam15.polyevents.helper
+
+import com.github.sdpteam15.polyevents.model.database.remote.Database.currentDatabase
+import com.github.sdpteam15.polyevents.model.database.remote.DatabaseConstant
+import com.github.sdpteam15.polyevents.model.entity.Event
+import com.github.sdpteam15.polyevents.model.entity.Item
+import com.github.sdpteam15.polyevents.model.entity.MaterialRequest
+import com.github.sdpteam15.polyevents.model.entity.MaterialRequest.Status.*
+import com.github.sdpteam15.polyevents.model.entity.Zone
+import com.github.sdpteam15.polyevents.model.observable.Observable
+import com.github.sdpteam15.polyevents.model.observable.ObservableList
+
+/**
+ * The goal of this objects is to create function to nested relation from the database
+ */
+object DatabaseHelper {
+
+    /**
+     * Delete an event and all its material request and event edits from the database
+     * @param e the event we want to delete
+     */
+    fun deleteEvent(e: Event) {
```

Comment:
So every time we delete an event we should use this method right?

Reply:
Yes, completely